portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,16 +54,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("viewer-comment");
-    String commentName = request.getParameter("viewer-comment-name");
-    String commentCombined = null;
-
     long timestamp = System.currentTimeMillis();
+    String name = null; 
+
+    if (userService.getCurrentUser()!=null){ //if someone is logged in 
```

Comment:
what is the intended behavior if there is no logged in user? Should this return an http 401 unauthorized response?